solvation_analysis/plotting.py

```diff
@@ -0,0 +1,294 @@
+
+import plotly
+import plotly.graph_objects as go
+import plotly.express as px
+import matplotlib
+
+import numpy as np
+import pandas as pd
+
+# single solution
+
+def plot_histogram(solution):
+    # histogram of what?
+    return
+
+def format_graph(fig, title, x_axis, y_axis):
+    # a formatting/styling function for each graph that deals with titles/labels
+    # make a legend
+    """
+
+    Parameters
+    ----------
+    fig :
+    title :
+    x_axis :
+    y_axis :
+
+    Returns
+    -------
+    fig : Plotly.Figure
+
+    """
+    fig.update_layout(xaxis_title_text=x_axis.title(), yaxis_title_text=y_axis.title(), title=title.title())
```

Comment:
I'd favor removing the `format_graph` function and just moving this to `plot_pairing`, `plot_coordination` etc. That will reduce the number of kwargs to `compare_solvent_dicts` and reduce the number of functions.